qemu/tests/cfg/hotplug_mem_repeat.cfg

```diff
@@ -3,44 +3,18 @@
 #    is support memory hotplug;
 - hotplug_memory_repeat:
     type = hotplug_mem_repeat
-    only Linux
     no Host_RHEL.m6
     no RHEL.5
+    no Windows..i386
+    no WinXP Win2000 Win2003 WinVista
     threshold = 210
```

Comment:
`threshold = 210`?  It is crazy,  it means only if `abs(guest_mem_size - vm_mem_size) > vm_mem_size * threshold`, case fail. refer:  https://github.com/avocado-framework/avocado-vt/blob/e4c4d73b8ec34c6386fcaa1f986767641ab48176/virttest/utils_test/qemu/__init__.py#L709